pithos/plugins/notify.py

```diff
@@ -50,25 +48,20 @@ def send_notification(self, window, *ignore):
         notification = Gio.Notification.new(song.artist)
         notification.set_body(song.title)
         if song.artUrl:
-            notification.set_icon(Gio.FileIcon.new(Gio.File.new_for_uri(song.artUrl)))
+            icon = Gio.FileIcon.new(Gio.File.new_for_uri(song.artUrl))
+        else:
+            icon = Gio.ThemedIcon.new_with_default_fallbacks('audio-x-generic-symbolic')
+        notification.set_icon(icon)
 
         if window.playing:
             notification.add_button(_('Pause'), 'app.pause')
         else:
             notification.add_button(_('Play'), 'app.play')
         notification.add_button(_('Skip'), 'app.next-song')
 
-        if self._last_id != song.trackToken:
-            self._app.withdraw_notification(self._last_id)
-
-        self._last_id = song.trackToken
-        self._app.send_notification(song.trackToken, notification)
+        self._app.send_notification('Pithos Song Change Notification', notification)
```

Comment:
The id isn't a user exposed value, this can be simplified to just our app-id or something that we know is unique.

Reply:
OK app id it is then